src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int bothside;
+    private int height;
+
+    public RightTriangle(String color, int height, int bothside) {
+        super(color);
+        this.height = height;
+        this.bothside = bothside;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: Right Triangle");
+        System.out.println("Color: " + getColor());
+        System.out.println("Both side length: " + bothside);
+        System.out.println("Height length: " + height);
```

Comment:
Consider creating a method to generate the common description for figures to avoid code duplication across different figure types. This method could be placed in the base class 'Figure' and called from the 'draw' method in subclasses.